lib/fizzy/execute.cpp

```diff
@@ -120,6 +151,9 @@ std::vector<TypeIdx> match_imports(const Module& module,
         case ExternalKind::Function:
             imported_function_types.emplace_back(import.desc.function_type_index);
             break;
+        case ExternalKind::Table:
```

Comment:
Now the default case cannot happen.

Reply:
Should I replace it with or assertion or better remove it?

Reply:
I would leave an assertion. I think this is UB to have an enum object with a value out of definition, but I'm not sure how to exactly handle that. Moreover, there is at least one more case like that with `SectionId`.

Reply:
`SectionId` switch in parser loop (and also switches to for `ExternalKind` in parser) are somewhat different case, because there default case can really happen when parsing and invalid module. Here I'll change it to assertion.